release-notes/major10/10.5.0/migration.md

```diff
@@ -2,6 +2,31 @@
 
 This page describes how to update Valtimo from the previous version to the current.
 
+* **Migrate form process 'Retrieve Document and link to Zaak' to process 'Link document to zaak'**
```

Comment:
This should be removed because this migration is already in this file.

Reply:
Fixed it. I thought it was two items since I had merge conflict in this file.

Reply:
Could you remove the changes to this file? We would like to keep the step-names

Reply:
Removed it

Reply:
Its still there...

Reply:
https://github.com/valtimo-platform/valtimo-documentation/pull/305/files it is actually, idk why the pr still shows it

Reply:
I just checked it. It really is still there